app/src/main/java/com/codesoom/assignment/controllers/SessionController.java

```diff
@@ -0,0 +1,37 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.application.AuthenticationService;
+import com.codesoom.assignment.dto.SessionResponseData;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * session에 대한 요청을 처리
```

Comment:
JavaDoc의 메인 디스크립션은 불특정 다수에게 공개되는 문서로 빌드될 수 있습니다. 문장을 완성해주세요.
```suggestion
 * session과 관련된 HTTP 요청을 처리합니다.
```